### PR TITLE
data(wb_gender): add wb gender statistics to meadow

### PR DESCRIPTION
**Pull request by lucasrodes**
Adding Meadow dataset for World Bank Gender Statistics.

### CODE REVIEW

File: etl/steps/data/meadow/wb/2022/wb_gender.py [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+    metadata_filename = "Gender_StatsSeries.csv"
+
+    def __init__(self) -> None:
+        log.info("Loading walden dataset...")
+        self.walden_ds = walden.Catalog().find_one("wb", "2022", "wb_gender")
+
+    @property
+    def columns_relevant(self) -> List[str]:
+        """Get list of relevant columns from data file."""
+        return list(self.columns_data_rename.keys()) + [str(i) for i in range(1960, 2022)]
```
Comment (pabloarosado) [inline etl/steps/data/meadow/wb/2022/wb_gender.py:34]: I'd recommend defining these kinds of parameters as global variables at the top, to be aware that they need to be changed by the next update.

File: etl/steps/data/meadow/wb/2022/wb_gender.py [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+
+    @property
+    def coluns_not_relevant(self) -> List[str]:
```
Comment (pabloarosado) [inline etl/steps/data/meadow/wb/2022/wb_gender.py:37]: Typo.

File: etl/steps/data/meadow/wb/2022/wb_gender.py [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+    for t in tables:
+        table = Table(t[0])
+        table.metadata.short_name = t[1]
+        ds.add(utils.underscore_table(table))
+    return ds
+
+
+def run(dest_dir: str) -> None:
+    """Run pipeline."""
+    WBGenderMeadowStep().run(dest_dir)
```
Comment (pabloarosado) [inline etl/steps/data/meadow/wb/2022/wb_gender.py:258]: I'd personally prefer to have the `run` function in a data step as explicit as possible. So, if anyone wants to quickly have an overview of what's done to the data, they should be able to check the content of `run` and get a sense of it. So, at the very least, I'd recommend moving the content of the `run` method of `WBGenderMeadowStep` into the global `run` function. This way you can still see:
```
# Load data and metadata
df, metadata = self.load_data_from_walden()
...
```
Feel free to ignore my suggestion if you disagree.

**Review by pabloarosado — APPROVED** [review]

File: etl/steps/data/meadow/wb/2022/wb_gender.py (hunk 3 above)
Comment (lucasrodes) [reply etl/steps/data/meadow/wb/2022/wb_gender.py:258]: makes sense, thanks!

File: etl/steps/data/meadow/wb/2022/wb_gender.py (hunk 1 above)
Comment (lucasrodes) [reply etl/steps/data/meadow/wb/2022/wb_gender.py]: makes sense!

**Comment by lucasrodes** [comment]
thanks for your review, @pabloarosado!

**Pull request merged**